### ISSUE TITLE
[BUG] match docstring example does not match the beginning of strings

### ISSUE THREAD

**Issue by beckernick** [rapidsai/custrings#271]
`match` says it will "Return array of boolean values where True is set if the specified pattern matches the beginning of the corresponding string."

The docstring example is not consistent with this description.

```
import nvstrings
s = nvstrings.to_device(["hello","there","world"])

print(s.match('h'))
Output:

[True, False, True]
```

**Comment by davidwendt** [comment]
Not seeing this issue in the current code.
```
import nvstrings
s = nvstrings.to_device(["hello","there","world"])
print(s.match('h'))
```
```
[True, False, False]
```

**Comment by beckernick** [comment]
Yep, we just need to update the docstring [here](https://github.com/rapidsai/custrings/blob/d94221b54fbe3c75fdee0281ae0a04c7c33a36b0/python/nvstrings.py#L1917)

**Comment by davidwendt** [comment]
Fixed with changes in #273
